dictionary.hh

```diff
@@ -312,6 +312,9 @@ public:
   vector< string > const & getDictionaryFilenames() noexcept
   { return dictionaryFiles; }
 
+  /// Get the main folder that contains the dictionary, without the ending separator .
```

Comment:
I added a comment here

Reply:
![image](https://user-images.githubusercontent.com/105986/231939678-cc4885ea-c594-419c-bade-d9952e9a2d5d.png)
the original code has this part

Reply:
Because they use the `QFileInfo::absolateDir`which returns a `QDir` then obtain the QString `absolutePath`.

The intermediate step is unncessary :sweat_smile: 

We just cut short with `QFileInfo::absolatePath -> QString`.

Reply:
The old code called a wrapper, then unwrap it :sweat_smile:  
![image](https://user-images.githubusercontent.com/20123683/231940351-c07b30fb-b474-4384-83b9-5bc335a837d6.png)

Reply:
QDir is not just a wrapper ,I think it return folder.  it change the value.

Reply:
The QFileInfo::absolutePath is enough to get the path of that folder.